### PR TITLE
2019 Azure Stack hybrid Profile.

### PR DESCRIPTION
**Pull request by sijuman**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: profile/2019-03-01-hybrid.json [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                        "publishingUsers",
+                        "recommendations",
+                        "runtimes",
+                        "sourceControls",
+                        "validate"
+            ]
+        }
+
+    },
+
```
Comment (bganapa) [inline profile/2019-03-01-hybrid.json]: can you please remove the extra lines and white spaces, just like other profile files?

Comment (jhendrixMSFT) [reply profile/2019-03-01-hybrid.json]: And please fix the indentation.

File: profile/2019-03-01-hybrid.json [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+            ],
+
+            "2015-07-01-preview":[
+                    "providerOperations"
+            ]
+        },
+
+        "microsoft.compute": {
+
+            "2017-12-01":[
```
Comment (bganapa) [inline profile/2019-03-01-hybrid.json]: this should be 2017-03-30 as we chatted

File: profile/2019-03-01-hybrid.json [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+                    "metricDefinitions",
+                    "metrics"
+            ],
+
+            "2017-05-01-preview": [
+                    "diagnosticSettings",
+                    "diagnosticSettingCategories"
+            ],
+
+            "2017-03-01-preview": [
```
Comment (bganapa) [inline profile/2019-03-01-hybrid.json]: api version for the event* resource types would be 2015-04-01, -preview is just being used for internal UX purposes

Comment (jhendrixMSFT) [reply profile/2019-03-01-hybrid.json]: Does that mean it shouldn't be included in the generated SDK profiles?

Comment (bganapa) [reply profile/2019-03-01-hybrid.json]: Yes, the insights team does not want the 2017* api version in the SDKs or tools. The api version got consumed, just for easy porting of the Ux, it did not add any new features in that api-version

**Review by bganapa — APPROVED** [review]

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**